solr_conf/conf/solrconfig.xml

```diff
@@ -11,10 +11,10 @@
   <directoryFactory name="DirectoryFactory" class="${solr.directoryFactory:solr.StandardDirectoryFactory}"/>
 
   <!-- solr lib dirs -->
```

Comment:
eh, nevermind, I'm thinking of the schema changes in HN2. I'd love to see some docs related to how all this solr config gets sorted out in different environments.